jsonnet/components/thanos-querier.libsonnet

```diff
@@ -70,6 +70,11 @@ function(params)
           resources: ['subjectaccessreviews'],
           verbs: ['create'],
         },
+        {
+          apiGroups: [''],
+          resources: ['pods'],
+          verbs: ['get', 'create'],
+        },
```

Comment:
I don't think that there's a need to add a new RBAC permission to the thanos-querier SA.

Reply:
Ah! Right, makes sense!